includes/classes/cache.php

```diff
@@ -57,7 +57,8 @@ public function sql_cache_is_expired($zf_query, $zf_cachetime): bool
         $zp_cache_name = $this->cache_generate_cache_name($zf_query);
         switch (SQL_CACHE_METHOD) {
             case 'file':
-                if (@filemtime(DIR_FS_SQL_CACHE . '/' . $zp_cache_name . '.sql') > (time() - $zf_cachetime)) {
+                $filename = DIR_FS_SQL_CACHE . '/' . $zp_cache_name . '.sql';
+                if (file_exists($filename) && filemtime($filename) > (time() - $zf_cachetime)) {
```

Comment:
I'm pretty sure that the `@` suppressor was originally used to accommodate cases where the server OS has forbidden making calls to filemtime causing PHP to croak.
I'm not sure if there's a straightforward way to programmatically pre-determine whether it will fail.
We "could" use `try/catch` exception handling if we want to avoid the `@`.

Reply:
Perhaps, but I'm more of the mind that the suppressor's use was that `filemtime` will throw a warning if the file doesn't exist.

Reply:
IMO that's the simplest way. And the code previously failed gracefully with the suppressor, so we're not changing what's worked before.